Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,109 @@
+"""
+NCL_sat_1.py
+===============
+This script illustrates the following concepts:
+   - Using 'astype' to unpack 'short' data
+   - Drawing line contours over a satellite map
+   - Changing the view of a satellite map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/slp.1963.nc"), decode_times=False)
+
+# Get lon/lat data from the 24th timestep
+U = ds.slp[24, :, :]
+
+# Translate short values to float values
+U = U.astype('float64')
+
+# Convert Pa to hPa data
+U = U*0.01
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_U = gvutil.xr_add_cyclic_longitudes(U, "lon")
+
+###############################################################################
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
+
+# Add land, coastlines, and ocean features
+ax.add_feature(cfeature.LAND, facecolor='lightgray')
+ax.add_feature(cfeature.COASTLINE, linewidth=.5)
+ax.add_feature(cfeature.OCEAN, facecolor='lightcyan')
+ax.add_feature(cfeature.BORDERS, linewidth=.5)
+ax.add_feature(cfeature.LAKES, facecolor='lightcyan', edgecolor='k', linewidth=.5)
+
+# Plot contour data
+p = wrap_U.plot.contour(ax=ax,
+                        transform=ccrs.PlateCarree(),
+                        linewidths=0.5,
+                        levels=30,
+                        cmap='black',
+                        add_labels=False)
+
+# Specify array of contour levels to be labeled- These values were found by setting 'manual'
+# argument in ax.clabel call to 'True' and then hovering mouse over desired location of
+# countour label to find coordinate (which can be found in bottom left of figure window)
+
+# low pressure contour levels- these will be plotted as a subscript to an 'L' symbol
+lowClevels = [(-3.825, 4.063), (1.725, 2.026), (1.663, 4.479)]
```

Comment:
I'd be hesitant to manually mark the coordinates of the low pressure areas. There should be a local minimum pressure in the data, so use that to find the exact latitude and longitude of the lowest pressure. You can then use that coordinate to draw the L.

Reply:
I tried doing this by using the "levels" argument in ax.clabel and setting it to the local minimum pressures ([972, 976, 960]). However, only one of the L labels showed up and it was off to the side of where it showed up in the NCL graph. Do you know why this might be?

Reply:
Are you sure those contour lines were drawn by `contour`? You can force those lines to be drawn by using the kwarg `levels` for `contour`. You can give it an array of the levels you want.

Reply:
Even when I set the contour "levels" kwarg to levels=[960, 972, 976], I still get this:

![polyg_19_limited](https://user-images.githubusercontent.com/36773444/85336672-dc73f300-b493-11ea-9d00-850edbab9995.PNG)

(This is an example of only plotting those levels)

Reply:
If that's the case then I think the solution you have is the best you can do.